src/query/queryBuilder.ts

```diff
@@ -166,16 +166,42 @@ function buildConditionExpressions<T extends object = AnyObject>(
         );
       } else if (operator === 'IN') {
         let expressionTemp = '';
+        const arraysOfContaingMax100Values = [];
+        let tempArray = [];
         condition.forEach((eachVal, index) => {
-          expressionAttributeValues[`${valueExpression}${index + 1}`] = eachVal;
-          if (!expressionTemp) {
-            expressionTemp = expressionTemp + `${valueExpression}${index + 1}`;
-          } else {
-            expressionTemp =
-              expressionTemp + `, ${valueExpression}${index + 1}`;
+          tempArray.push(eachVal);
+          if (
+            (index + 1) % 100 === 0 ||
+            (condition.length < 100 && index + 1 === condition.length)
+          ) {
+            arraysOfContaingMax100Values.push(tempArray);
```

Comment:
Use lodash to get this (arraysOfContaingMax100Values).
lodash is already in the dependencies
`lodash.chuck(arr, size)`

Reply:
Name of const is too generic. Please rename this `arraysOfContaingMax100Values` to -
```suggestion
            conditionChunksOf100.push(tempArray);
```